### ISSUE TITLE
Languages not alphabetically sorted

### ISSUE THREAD

**Issue by endorama** [damcclean/hacktoberfest-finder#209]
👋 Hello! First of all thank for this repo :tada: Searching issues is always a pain during Hacktoberfest so thank you!

While using it I had some issues with the sorting of the languages :sweat: I was expecting them to be alphabetically ordered, but they are not.

I've seen the `topLanguages` in `app.js`

https://github.com/damcclean/hacktoberfest-finder/blob/ec8396eaa59a3d29a68edbae88bbd964970ba10c/src/js/app.js#L9-L10

but it seems like `topLanguages` is not used anymore. `allLanguages` is used but is not sorted (I'm not sure of the reason).

## How to reproduce

Just go to the website and open the Languages dropdown.

## What I expected

Languages to be alphabetically sorted (or a better distinction between "top" and lower languages").

**Comment by damcclean** [comment]
Hey!

Glad I could be of help during Hacktoberfest! Most of the site has been built out by [contributors](https://github.com/damcclean/hacktoberfest-finder/blob/master/CONTRIBUTORS.md), I've really just maintained the project and kept it going. 😅

That's actually a good point. It would be a good idea for the languages to be sorted into alphabetical order.

To be entirely honest, I'm not sure what `topLanguages` does/used to do. The JavaScript in this project needs cleaned up at some point. I'll probably do it when I have some spare time.

At the start of the month, we moved the languages to its own file ([`src/js/languages.js`](https://github.com/damcclean/hacktoberfest-finder/blob/master/src/js/languages.js)) to help tidy stuff up a little (I'm sure there was another reason behind it though)

I've tagged this issue as 'Hacktoberfest' for anyone to go ahead and fix the sorting issue, while following the [contributors guide](https://github.com/damcclean/hacktoberfest-finder/blob/master/CONTRIBUTING.md).
